integration-tests/oauth2/src/test/groovy/com/okta/spring/tests/oauth2/implicit/ImplicitRemoteValidationGroupIT.groovy

```diff
@@ -38,4 +42,19 @@ class ImplicitRemoteValidationGroupIT extends ApplicationTestRunner {
         .then()
             .body(Matchers.equalTo("Everyone has Access: joe.coder@example.com"))
     }
+
+    @Test
+    void validateUserAgent() {
+
+        ResourceBundle versions = ResourceBundle.getBundle("versions")
+        def springIntegrationVersion = versions.getString("project.version")
+        def sdkVersion = versions.getString("okta.sdk.version")
+
+        // discovery
+        wireMockServer.verify(
+                newRequestPattern(RequestMethod.GET, urlMatching("/oauth2/default/.well-known/openid-configuration"))
+                    .withHeader("User-Agent", containing("okta-spring-security/${springIntegrationVersion}"))
+                    .withHeader("User-Agent", containing("okta-sdk-java/${sdkVersion}"))
```

Comment:
In our other integrations, we're using `X-Okta-User-Agent-Extended` instead of `User-Agent`. However, this isn't captured by our wiki.

@robertjd ^

Reply:
X-Okta is only used by front ends.  This matches the backend (and SDKs)